src/value.rs

```diff
@@ -282,6 +282,19 @@ impl Value {
 
                 Some(evaled_arg.eval_to_rc(Rc::clone(environment)))
             }
+            Value::PersistentListMap(map) => {
+                if args.len() != 1 {
+                    return Some(Rc::new(Value::Condition(format!(
+                        "Wrong number of arguments (Given: {}, Expected: 1)",
+                        args.len()
+                    ))));
+                }
+                let arg_rc_values = PersistentList::iter(args)
+                    .map(|rc_arg| rc_arg)
+                    .collect::<Vec<Rc<Value>>>();
+                let key = arg_rc_values.get(0).unwrap();
```

Comment:
Do we want to eval the arg (i.e. `key`) before `map.get(key)`?

Reply:
After working on supporting `Value::Keyword` in function position, I believe we _do_ want to eval before attempting to get from the map (i.e. eval _has not_ already happened, at this point).